src/flake8_bugbear/plugins/duplicate_exceptions.rs

```diff
@@ -34,7 +34,7 @@ fn duplicate_handler_exceptions<'a>(
             if seen.contains_key(&call_path) {
                 duplicates.insert(call_path);
             } else {
-                seen.insert(call_path, type_);
+                seen.entry(call_path).or_insert(type_);
```

Comment:
Lol I was just trying to get this to work... Seems like a false-positive with Clippy, since the autofix fails?